### PR TITLE
New package: app.vleer version 1.0.0-beta.3

### PR DESCRIPTION
**Pull request by 0PandaDEV**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/152224)

### CODE REVIEW

**Comment by 0PandaDEV** [comment]
@microsoft-github-policy-service agree

**Comment by 0PandaDEV** [comment]
how do i test the file when i execute `winget validate --manifest manifests\a\app\vleer\1.0.0-beta.3\app.vleer.yaml` i just get a error

**Comment by vikingnope** [comment]
@0PandaDEV, to test the package it needs to be your full path so it should include C:/etc..

**Comment by 0PandaDEV** [comment]
ok but `winget validate --manifest "C:\Users\pandadev\Desktop\vleer\manifests\a\app\vleer\1.0.0-beta.3\app.vleer.yaml"`
also returns 
```
Manifest validation failed.
Manifest Error: The multi file manifest is incomplete.A multi file manifest must contain at least version, installer and defaultLocale manifest. File: app.vleer.yaml
```

**Comment by vikingnope** [comment]
the location should not contain the yaml, yours should be as follows: `winget validate --manifest "C:\Users\pandadev\Desktop\vleer\manifests\a\app\vleer\1.0.0-beta.3`

**Comment by 0PandaDEV** [comment]
Ok I verified it works, I tested it with `validate` and `install`

**Comment by stephengillie** [comment]
Verify: 
image

File: manifests/a/app/vleer/1.0.0-beta.3/app.vleer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: app.vleer
+PackageVersion: 1.0.0-beta.3
```
Comment (stephengillie) [inline manifests/a/app/vleer/1.0.0-beta.3/app.vleer.installer.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```
Please update file name and path to match this change.

File: manifests/a/app/vleer/1.0.0-beta.3/app.vleer.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: app.vleer
+PackageVersion: 1.0.0-beta.3
```
Comment (stephengillie) [inline manifests/a/app/vleer/1.0.0-beta.3/app.vleer.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```
Please update file name and path to match this change.

File: manifests/a/app/vleer/1.0.0-beta.3/app.vleer.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: app.vleer
+PackageVersion: 1.0.0-beta.3
```
Comment (stephengillie) [inline manifests/a/app/vleer/1.0.0-beta.3/app.vleer.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```
Please update file name and path to match this change.

**Review by stephengillie — COMMENTED** [review]
Hi @0PandaDEV,

If possible, the `PackageVersion` should match the Registry. 
- If the package shows `-beta.3` then this can be the `PackageVersion` if `DisplayVersion: 1.0.0` is added under `AppsAndFeaturesEntries`. Because these might be the only bits on the PC for our package manager to match the installed package against the repository manifest data.

**Comment by 0PandaDEV** [comment]
Regarding the path error that is because the app is compiled with rust which does not allow characters other than numbers and points in the version name so it shows as 1.0.0 in windows, but it's not the release 1.0.0 it's a beta because the app has not yet been released.

File: manifests/a/app/vleer/1.0.0-beta.3/app.vleer.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: app.vleer
+PackageVersion: 1.0.0-beta.3
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{8905B882-D647-4593-946E-5537499CCFBA}'
+Installers:
```
Comment (stephengillie) [inline manifests/a/app/vleer/1.0.0-beta.3/app.vleer.installer.yaml:9]:
```suggestion
AppsAndFeaturesEntries:
  DisplayVersion 1.0.0
Installers:
```

Oh, then a `DisplayVersion` is needed. This way, the package manager will show `1.0.0-beta.3` when listing and describing installed packages, but will know to match with `1.0.0` in the Registry.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-05-06T21:56:25.6625051Z ##[error]Manifest Validation Failed
 2024-05-06T21:56:30.7080903Z ##[section]Finishing: Validate Manifest

(Automated response - build 880.)

File: manifests/a/app/vleer/1.0.0-beta.3/app.vleer.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: app.vleer
+PackageVersion: 1.0.0-beta.3
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{8905B882-D647-4593-946E-5537499CCFBA}'
+AppsAndFeaturesEntries:
+  DisplayVersion 1.0.0
```
Comment (stephengillie) [inline manifests/a/app/vleer/1.0.0-beta.3/app.vleer.installer.yaml:10]:
```suggestion
- DisplayVersion: 1.0.0
```
I provided invalid syntax, apologies.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**